chess-AI/experimental-work/nn_layout.py

```diff
@@ -99,22 +125,3 @@ def forward(self, x):
 a value head of one scalar evaluation number.
 
 """
-def main():
-    model = PlayNetwork()
-    policy, value = model(torch.randn(1, 119, 8, 8))
-    policy = policy.reshape(8, 8, 73)
-    value = value.item()
-    print(policy)
-    print(value)
-
-    import chess
-    board = chess.Board()
-    policy_converter = PlayNetworkPolicyConverter()
-
-    move_values = policy_converter.find_value_of_all_legal_moves(policy, board)
-    for move, value in move_values.items():
-        print(f"{move}: {value}")
-
-
-if __name__ == "__main__":
-    main()
```

Comment:
You can leave this code - it won't be run when importing stuff from `nn_layout` and it's useful as a reference.